packages/shared/src/components/cards/AdAttribution.tsx

```diff
@@ -5,14 +5,17 @@ import { Ad } from '../../graphql/posts';
 export type AdAttributionProps = {
   ad: Ad;
   className?: string;
+  typoClassName?: string;
```

Comment:
It's better to opt for `className` as object, you can see some existing implementations already or @sshanzel can support you as well.